packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -79,7 +80,24 @@ export class AtomicBreadbox implements InitializableComponent {
 
   private breadcrumbShowLessFocus?: FocusTargetController;
 
+  /**
+   * This prop allows you to control the display of a path by
+   * limiting the number of visible values in the path.
+   *
+   * If the path size is equal to or less than the pathLimit, all values in
+   * the path will be displayed without truncation.
+   *
+   * If the path size exceeds the pathLimit, it will truncate the path by
+   * replacing the middle values with ellipses ('...').
+   *
+   * Minimum: `3`
+   * Maximum: `10`
```

Comment:
Is there a reason for that value? Can't we have a higher maximum?

Reply:
Minimum should be `1` because that's just a logical/easily understandable limit.

For maximum, I'd say we should support infinite length if an implementer specifically configure it that way.

Yeah, it would break the rendering/would look silly with hundreds of values, but 🤷

The upper limit of 10 here is too arbitrary. 
We can add a warning I guess, that any big value can cause weird stuff to happen in the template.

And, I'd like if we did some manual test to check what happens if we mock a value with hundreds of values, and try to display them all.

Just to get an idea about how does our standard result templates sections behave.